docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -15,6 +15,7 @@ kyma alpha deploy [flags]
 ## Options
 
 ```bash
+  -a, --atomic                    Use atomic deployment (rollback any component which could not be successfully installed). (default true)
       --cancel-timeout duration   Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client. (default 15m0s)
   -c, --components string         Path to the components file. (default "workspace/installation/resources/components.yaml")
```

Comment:
```suggestion
  -c, --components string         Path to the components file (default: "workspace/installation/resources/components.yaml")
```

Reply:
Good input, but we need to handled that decoupled form this PR as it requires a change in the doc-generator logic.